target_chains/fuel/contracts/pyth-contract/src/pyth_merkle_proof.sw

```diff
@@ -37,9 +37,9 @@ fn node_hash(child_a: Bytes, child_b: Bytes) -> Bytes {
 
 pub fn validate_proof(
     encoded_proof: Bytes,
-    leaf_data: Bytes,
     ref mut proof_offset: u64,
     root: Bytes,
+    leaf_data: Bytes,
```

Comment:
drive-by: standardizing it to be the same order as the [solidity counterpart](https://github.com/pyth-network/pyth-crosschain/blob/main/target_chains/ethereum/contracts/contracts/libraries/MerkleTree.sol#L47-L52)